File: Python/백준/Silver/1476. 날짜 계산/날짜 계산.py
import sys
E, S, M = map(int, sys.stdin.readline().split())
E -= 1
S -= 1
M -= 1
# E 1 ~ 15, S 1 ~ 28, M 1 ~ 19
ans = 0
while True:
    if E == ans % 15 and S == ans % 28 and M == ans % 19:
        print(ans + 1)
        break
    else:
        ans += 1

File: Java/백준/Silver/1476. 날짜 계산/날짜 계산.java
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.BufferedReader;
import java.io.OutputStreamWriter;
import java.io.BufferedWriter;
import java.util.*;

public class Main {
    static InputStreamReader reader = new InputStreamReader(System.in);
    static BufferedReader br = new BufferedReader(reader);

    static OutputStreamWriter writer = new OutputStreamWriter(System.out);
    static BufferedWriter bw = new BufferedWriter(writer);

    public static void main(String[] args) throws IOException {
        String inp[] = br.readLine().split(" ");
        int E = Integer.parseInt(inp[0]) - 1, S = Integer.parseInt(inp[1]) - 1, M = Integer.parseInt(inp[2]) - 1;

        int targetNum = S;
        while (true) {
            int tE = targetNum % 15;
            int tM = targetNum % 19;
            if (tE == E && tM == M) break;
            targetNum += 28;
        }
        bw.write(Integer.toString(targetNum + 1) + "\n");

        bw.flush();
        bw.close();
    }
}